tutorials/tracing/manual_instrumentation_tutorial.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>Suggestion: Show the dataframes so people can see what data they're working with.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>